lib/overcommit/hook_context/post_commit.rb

```diff
@@ -0,0 +1,20 @@
+module Overcommit::HookContext
+  # Contains helpers related to contextual information used by post-commit
+  # hooks.
+  class PostCommit < Base
+    # Get a list of files that were added, copied, or modified in the last
+    # commit. Renames and deletions are ignored, since there should be nothing
+    # to check.
+    def modified_files_last_commit
```

Comment:
I would prefer this to just be called `modified_files` to match that of the `PreCommit` hook context. The goal with these context classes was to provide a somewhat common API across all types of hooks where it made sense to do so. In this case, it makes sense that `modified_files` would only apply to the last commit when in a `post-commit` context.